src/Conf.ml

```diff
@@ -957,6 +968,12 @@ let validate () =
 ;;
 parse info validate
 
+let to_absolute file =
+  Filename.(if is_relative file then concat (Unix.getcwd ()) file else file)
```

Comment:
nit: It would probably be better to switch all filename manipulation to Fpath.

Reply:
I don't understand how this change disrupts the CWD handling in test.sh, do you?